### PR TITLE
Milestone 1: Project complete

### PR DESCRIPTION
**Pull request by peterrobert**
project ready for review

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
@peterrobert 
### This is impressive, good job so far, however, there are few changes you need to make for the project to be approved
- Write a more descriptive pull request showing what you have done on the project [check this guide for more info](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
- write a more detailed readme file explaining what the project is all about, technologies, license, contributors, who I can install the project [check this guide](https://github.com/microverseinc/readme-template)
- Make sure your project passes linting both HTML and CSS [check this guide](https://github.com/microverseinc/linters-config/tree/master/css)
- Your navigation is still off from the original design, no logo, the gradient is still off and the search 
navigation

- Always remember you can ask for help from slack, your team, mentor and or any TSE
### Once you make all these changes, resubmit your project for a review. I wish you all the best

**Comment by peterrobert** [comment]
**Building-with-Backgrounds-and-Gradients Milestone 1: Complete Project**

In this milestone, I have completed the following:

- Created and validated the HTML mark up.

- Created and validated the CSS mark up.

- Added font-awesome and used some of its elements on my mark up

- Added stickler in order to fix common CSS issues

File: index.html [hunk 1]
```diff
@@ -6,16 +6,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Document</title>
+    <!-- ====fonts ==== -->
```
Comment (euqueme) [inline index.html]: Delete commented code

File: index.html [hunk 2]
```diff
@@ -6,16 +6,85 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu&display=swap" rel="stylesheet">
     <!-- === links === -->
```
Comment (euqueme) [inline index.html]: **Delete commented code**

File: index.html [hunk 3]
```diff
@@ -6,16 +6,85 @@
...
+                <h2>every one has something to share.see how people are using ipad in their unique ways to add
+                    to the worlds story </h2>
+                <h3> <a href="">watch the "your verse" film and explore stories</a> </h3>
+            </hgroup>
+            <!-- <i class="fa fa-play-circle-o"  aria-hidden="true"></i> -->
 
+        </section> <!-- .main-body -->
+    </main>
 
-    <!-- ==== font awesome cdn ===== -->
```
Comment (euqueme) [inline index.html:17]: Delete commented code

File: index.html [hunk 4]
```diff
@@ -6,16 +6,85 @@
...
-    <script src="https://use.fontawesome.com/da489ccde9.js"></script>
+    <!-- ===== footer section === -->
```
Comment (euqueme) [inline index.html]: Delete commented code

File: style.css [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+}
+
+.main-navigation ul li {
+  box-shadow: 0 1px 2px rgb(0, 0, 0, 0.3);
+  padding-left: 35px;
+  padding-right: 35px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  text-transform: capitalize;
+  background-image: linear-gradient(#333, #5c5c5c);
```
Comment (euqueme) [inline style.css]: [Optional] try this gradient instead
background-image: linear-gradient(#9c9c9c 20%,#5c5c5c, #4c4c4c);

File: index.html [hunk 6]
```diff
@@ -6,16 +6,85 @@
...
+                <li>
+                    <img src="images/5.jpg" alt="">
+                </li>
+            </ul>
+        </div>
+
+        <div class="footer-details">
+            <div class="shop-online">
+                <h5>Shop the Apple Online Store (1-800-MY-APPLE), visit an Apple Retail Store, or find a reseller.</h5>
+                <h5>Copyright © 2014 Apple Inc. All rights reserved. Terms of UsePrivacy Policy</h5>
```
Comment (euqueme) [inline index.html]: The font-weight of these lines should be less than 500

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) 

### Hello @peterrobert  :bulb: 
### I'm really impressed by the good work you've done so far :clap: thank you for making the changes the previous TSE requested, there are still a couple of mandatory requirements left before we can give you the approval:

- [ ]  The text in your footer shouldn't be bold

 **General**

- [ ] Comments: It’s better to delete commented code

[OPTIONAL] your navbar grid is darker than the original maybe you could play with some different combinations to achieve a more similar look (I have left an example in the commented code for you to try.)

#### I have commented on the lines of code that need to be fixed but its been a blast to review this project. :clinking_glasses:

### Please :pray: correct these changes and submit back for a code review when you are done :rocket:

#### Happy Coding! :woman_technologist:

STATUS OF THE PROJECT: REQUESTED CHANGES :traffic_light:

**Comment by peterrobert** [comment]
All changes have been done as asked.

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE) :woman_technologist:

### Hii @peterrobert :handshake: 

I downloaded your project and I notice you made all the changes :raised_hands:   but the Live Demo doesn't reflect them :thinking: 
image

### Please correct this :memo:  and submit the project right away no need to ask for a code review anymore

STATUS OF THE PROJECT: APPROVED :green_apple:

**Pull request merged**